pkg/frontend/openshiftclustercredentials_post.go

```diff
@@ -60,5 +60,10 @@ func (f *frontend) _postOpenShiftClusterCredentials(ctx context.Context, r *http
 
 	doc.OpenShiftCluster.Properties.ServicePrincipalProfile.ClientSecret = ""
 
-	return json.MarshalIndent(converter.ToExternal(doc.OpenShiftCluster), "", "    ")
+	ext := converter.ToExternal(doc.OpenShiftCluster)
```

Comment:
I was wondering if we could check `if f.apis[vars["api-version"]].OpenShiftClusterCredentialsConverter() == nil` around line 33 instead?  It would save entering this function at all.

Reply:
@jim-minter oh, it's much better than what I did initially! Saves us a couple of db calls before we error and makes mocking a bit easier.

I updated the PR. Please take another look.